### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by y8j31**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2020-2021 The University of Manchester
```
Comment (mithro) [inline LICENSE]: BTW You shouldn't change this file.

**Review by robtaylor — CHANGES_REQUESTED** [review]
Please remove additional copyright line - as discussed copyrights belong in the files that the copyright holders have created.

File: LICENSE (hunk 1 above)
Comment (y8j31) [reply LICENSE]: Hi Mithro, do you mean I should keep "Copyright [yyyy] [name of copyright owner]" as what it was?

**Review by y8j31 — COMMENTED** [review]
The LICENSE file has been edited and the copyright line has returned to what it was.

File: LICENSE (hunk 1 above)
Comment (mithro) [reply LICENSE]: Yes, correct :-)

**Review by robtaylor — APPROVED** [review]

**Pull request merged**